buildbot-host/buildmaster/master.cfg

```diff
@@ -604,6 +607,22 @@ factories.update(
 )
 del factory
 
+# openvpn3 smoketest
+factory = util.BuildFactory()
+factory = openvpn3AddCommonLinuxStepsToBuildFactory(factory, combo)
```

Comment:
What is `combo` here supposed to mean. We are not in a `for` loop, are we? I know you only copied this, but still would like to clean this up.

Reply:
You're correct - copy and paste error. Not really sure what the implications are to be honest, but I'll clean it and the other places up.